pkg/service/service.go

```diff
@@ -863,7 +863,17 @@ func (s *service) SetDefaultUserPipelineReleaseByID(ctx context.Context, ns reso
 	return nil
 }
 
-func (s *service) triggerPipeline(ctx context.Context, ownerPermalink string, recipe *datamodel.Recipe, pipelineId string, pipelineUid uuid.UUID, pipelineInputs []*structpb.Struct, pipelineTriggerId string, returnTraces bool) ([]*structpb.Struct, *pipelinePB.TriggerMetadata, error) {
+func (s *service) triggerPipeline(
+	ctx context.Context,
+	ownerPermalink string,
+	recipe *datamodel.Recipe,
+	pipelineId string,
+	pipelineUid uuid.UUID,
+	pipelineReleaseId string,
+	pipelineReleaseUid string,
```

Comment:
I suggest that we use `uuid.UUID` type in the function params

Reply:
will do